### PR TITLE
Newline after parenthesis if two arguments or more

### PR DESCRIPTION
**Pull request by andersem**
Will format this:

```ts
const extractAccountIdFromPayload = async (paymentProvider: SettlementProvider, allObjects: ParsedAttachment[]): Promise<string | undefined> => {
    return `hithere ${paymentProvider} ${allObjects}`;
};
```

to this:

```ts
const extractAccountIdFromPayload = async (
    paymentProvider: SettlementProvider, allObjects: ParsedAttachment[]
): Promise<string | undefined> => {
    return `hithere ${paymentProvider} ${allObjects}`;
};
```

and this:

```ts
const normalMap = async (email: ParsedMail,
    paymentProvider: SettlementProvider,
    emailDate: Date,
    originKey: string,
): Promise<SettlementFiles[]> => {
    return [];
};
```

to this:

```ts
const normalMap = async (
    email: ParsedMail,
    paymentProvider: SettlementProvider,
    emailDate: Date,
    originKey: string,
): Promise<SettlementFiles[]> => {
    return [];
};
```

https://eslint.org/docs/rules/function-paren-newline

### CODE REVIEW

**Comment by frodeaa** [comment]
I would preferred if we used `minItems=1` as it would for `extractAccountIdFromPayload` introduce less changes if a new parameter is added to the function

**Review by frodeaa — APPROVED** [review]

**Comment by magnusjt** [comment]
I would prefer personal judgement per case on this one. Forcing it to newline makes one line into 3+ lines. I prefer fewer lines when possible, especially when they convey just one concept. As always, just my opinion.

**Comment by andersem** [comment]
I think maybe `multiline-arguments` is better, that way you decide if you want to split into multiple lines, but when it is split, it will break between each argument.

**Comment by sondresj** [comment]
> 
> 
> I think maybe `multiline-arguments` is better, that way you decide if you want to split into multiple lines, but when it is split, it will break between each argument.

Agreed, this is the best option IMO

File: default.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ module.exports = {
         "space-unary-ops": ["error", { "words": true, "nonwords": false }],
         "object-property-newline": ["error", { "allowAllPropertiesOnSameLine": true }],
         "space-in-parens": ["error", "never"],
+        "function-paren-newline": ["error", { "minItems": 2 }],
```
Comment (sondresj) [inline default.js]:
```suggestion
        "function-paren-newline": ["error", "multiline-arguments"],
```

**Review by sondresj — CHANGES_REQUESTED** [review]
If ya'll agree with this

**Review by sondresj — APPROVED** [review]

**Pull request merged**